### PR TITLE
[Doc][Task Plugin] Fix MLflow task plugin doc 

### LINKED ISSUE #11904: [Doc][Task Plugin] Fix MLflow task plugin doc
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/dolphinscheduler/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

- Fix the list format
- remove the todo list

### Documentation Links

https://dolphinscheduler.apache.org/en-us/docs/dev/user_doc/guide/task/mlflow.html

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by jieguangzhou**
## Purpose of the pull request

close: #11904

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

### CODE REVIEW

File: docs/docs/en/guide/task/mlflow.md [hunk 1]
```diff
@@ -13,19 +13,14 @@ MLflow task plugin used to execute MLflow tasks，Currently contains MLflow Proj
...
-    - [x] Docker: Run the container after packaging the docker image
-    - [x] Docker Compose: Use docker compose to run the container, it will replace the docker run above
-    - [ ] Seldon core: Use Selcon core to deploy model to k8s cluster
-    - [ ] k8s: Deploy containers directly to K8S
-    - [ ] MLflow deployments: Built-in deployment modules, such as built-in deployment to SageMaker, etc
-- [ ] Model Registry
-    - [ ] Register Model: Allows artifacts (Including model and related parameters, indicators) to be registered directly into the model center
+- MLflow Projects
+    - BasicAlgorithm: contains LogisticRegression, svm, lightgbm, xgboost
+    - AutoML: AutoML tool，contains autosklean, flaml
```
Comment (EricGao888) [inline docs/docs/en/guide/task/mlflow.md]:
```suggestion
    - AutoML: AutoML tool, contains autosklean, flaml
```

File: docs/docs/zh/guide/task/mlflow.md [hunk 2]
```diff
@@ -12,19 +12,14 @@ MLflow 组件用于执行 MLflow 任务，目前包含Mlflow Projects, 和MLflow
...
-  - [x] MLFLOW: 直接使用 `mlflow models serve` 部署模型。
-  - [x] Docker: 打包 DOCKER 镜像后部署模型。
-  - [x] Docker Compose: 使用Docker Compose 部署模型，将会取代上面的Docker部署。
-  - [ ] Seldon core: 构建完镜像后，使用Seldon Core 部署到k8s集群上, 可以使用Seldon Core的生成模型管理能力。
-  - [ ] k8s: 构建完镜像后， 部署到k8s集群上。
-  - [ ] MLflow deployments: 内置的允许MLflow 部署模块, 如内置的部署到Sagemaker等。
-- [ ] Model Registry
-  - [ ] Register Model: 注册相关工件(模型以及相关的参数，指标)到模型中心
+- MLflow Projects
+  - BasicAlgorithm: 基础算法，包含LogisticRegression, svm, lightgbm, xgboost
```
Comment (EricGao888) [inline docs/docs/zh/guide/task/mlflow.md]:
```suggestion
  - BasicAlgorithm: 基础算法，包含LogisticRegression，svm，lightgbm，xgboost
```

File: docs/docs/zh/guide/task/mlflow.md [hunk 3]
```diff
@@ -12,19 +12,14 @@ MLflow 组件用于执行 MLflow 任务，目前包含Mlflow Projects, 和MLflow
...
+  - AutoML: AutoML工具，包含autosklean, flaml
```
Comment (EricGao888) [inline docs/docs/zh/guide/task/mlflow.md]:
```suggestion
  - AutoML: AutoML工具，包含autosklean，flaml
```

**Review by EricGao888 — CHANGES_REQUESTED** [review]
LGTM except the punctuations.

File: docs/docs/en/guide/task/mlflow.md (hunk 1 above)
Comment (jieguangzhou) [reply docs/docs/en/guide/task/mlflow.md]: I've fixed it

**Comment by jieguangzhou** [comment]
@EricGao888 PTAL, thanks.

**Review by EricGao888 — APPROVED** [review]
LGTM

**Pull request merged**